generator/booking_dataset.cpp

```diff
@@ -55,21 +71,61 @@ BookingDataset::Hotel::Hotel(string const & src)
 
 ostream & operator<<(ostream & s, BookingDataset::Hotel const & h)
 {
+  s << fixed << setprecision(7);
```

Comment:
Not good to modify s state, like precision. Seems ok to me, as I thought that when one wants to put something to ostream, he should set up ostream correctly. But if it possible to roll-back ostream state, it would be nice.